Chapters/1-5-Chapter5.md

```diff
@@ -0,0 +1,155 @@
+## Chapter 5
+
+The city was carved from the body of an old mountain, curled upward as if the land itself was an ocean wave. Massive veins of white marble meandered between gray, banded rock, and striped through the roads and walls like an unmoving spectre casting shadows through time. Every building faced proudly toward Ara, carefully reflecting every glint of orange and red and infrared. High upon the mountain within the largest vein of marble sat the senate, with its crown, a the gilded halo of the Palinian circle. I imagined being one of the people that carved out their piece of the mountain to call home. _It must have taken ages_.
```

Comment:
> Every building faced proudly toward Ara, carefully reflecting every glint of orange and red and infrared.

Is it a weird mirrored solar power situation?

Reply:
umm yea sorta. I think I get into this later.